docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -159,7 +176,6 @@ compile these plugins from [source](https://github.com/gravitational/teleport-pl
 </TabItem>
 </Tabs>
 
-
 Run `./install` in from 'teleport-slack' or place the executable in the appropriate `/usr/bin` or `/usr/local/bin` on the server installation.
```

Comment:
Awesome, the `wget` fix above downloads a much smaller binary and `./install` can be run without issues that way.

Since we also provide the opportunity to download the source, the change in this PR may still be worth stating here since the install steps slightly vary in terms of paths and such: https://github.com/gravitational/teleport-plugins/pull/228 (the PR itself should probably be closed).

Reply:
There should be no difference in binary size between `curl` and `wget`, can you share a bit more info?

Reply:
Missing `-o` flag for `curl` still got 404 errors when going directly to the link as well. Anyway, resolved now. We should use `wget`.